### ISSUE TITLE
can not add one more  

### ISSUE THREAD

**Issue by Life-Think** [jashephe/Gmail-Notifier#23]
Hi, sir ! The app is very perfect for me. Thank U very much, But I have two Google Accounts, So I hope u can add the function！Thx!

**Comment by jashephe** [comment]
Hi!  Support for multiple accounts has been on the to-do list for a while now, and will most likely be incorporated in the next major release.  I haven't had as much time as I would have liked to work on this project in the last few months, so I can't give an ETA, I'm afraid.  In #12 I mention that gtm-oauth2 is the problem, but I've already managed to write a semi-functional replacement in Swift, so the upstream would no longer be considered blocking.  Also, I'd like to use Swift to write the next version, but it's been pretty unstable when I've tried it (I kept getting compiler crashes with my most recent build), so it still might be a while until we get a new version.
